### ISSUE TITLE
Error at Google Drive sync

### ISSUE THREAD

**Issue by Lukentui** [swiftyapp/swifty#18]
```
Error: invalid_grant
    at t.Gaxios.request (C:\Users\Lukentui\AppData\Local\Programs\Swifty\resources\app.asar\main\index.js:8:8812)
    at processTicksAndRejections (internal/process/task_queues.js:85:5)
    at async u.getTokenAsync (C:\Users\Lukentui\AppData\Local\Programs\Swifty\resources\app.asar\main\index.js:3:20676)
```

### Steps to reproduce

1. Open Settings
2. Click "Connect your Google Drive"
3. Login and permit access to Google Account(Click "Allow")
4. Catch exception :D

---

Swifty 0.4.1
Electron 7.0.0
win32 10.0.18362
Locale: ru

**Comment by alchaplinsky** [comment]
Related #19

**Comment by alchaplinsky** [comment]
Should be fixed in recent release [v0.4.4](https://github.com/swiftyapp/swifty/releases/tag/v0.4.4)
